plugins/bufbuild/connect-kotlin/v0.1.4/buf.plugin.yaml

```diff
@@ -4,7 +4,7 @@ plugin_version: v0.1.4
 source_url: https://github.com/bufbuild/connect-kotlin
 description: Idiomatic gRPC & Connect RPCs for Kotlin.
 deps:
-  - plugin: buf.build/protocolbuffers/java:v22.2
+  - plugin: buf.build/protocolbuffers/kotlin:v22.2
```

Comment:
Just to make sure: the transitive dependency with the java plugin gets applied here, right?

Reply:
Correct.